k8s/cms/kustomize/cms/overlays/common/oathkeeper/process_template.sh

```diff
@@ -8,9 +8,11 @@
 echo "Adding curl and jq"
 apk --no-cache add curl jq
 
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```

Comment:
Need to add this to have compatibility with intel machine as well. 

UNAME=$( uname -m)
case $UNAME in
   x86_64)
     ARCH="amd64"
     ;;
   aarch64)
     ARCH="arm64"
     ;;
   *)
     ARCH="amd64"
     ;;
esac

Reply:
Done. Note that I removed the `*)` case as it causes `arm64` to get incorrectly converted to `amd64`.